### ISSUE TITLE
[Linux] Low res icons, deformed in several places

### ISSUE THREAD

**Issue by Porkepix** [Manuel-777/MTG-Arena-Tool#948]
As said by the title, on Linux the icon looks low-res and deformed on several places, as visible on the screenshots below:

Top bar in Gnome
![Screenshot from 2020-03-24 19-03-31](https://user-images.githubusercontent.com/709500/77462213-189ba700-6e04-11ea-8227-d135bcb61abf.png)

Expose view in Gnome
![Screenshot from 2020-03-24 19-03-51](https://user-images.githubusercontent.com/709500/77462238-25b89600-6e04-11ea-8680-bfba43ffb088.png)

Alt-tab view in Gnome
![Screenshot from 2020-03-24 19-05-44](https://user-images.githubusercontent.com/709500/77462264-2fda9480-6e04-11ea-914d-1b7148cb90a4.png)

I might test later on on different Window Manager/Desktop Environment such as KDE.

**Comment by Manuel-777** [comment]
Can only test Manjaro Gnome at the moment, but the icon looks good on default settings, is this happening still?

**Comment by Porkepix** [comment]
Yes it's still happening, and my Linux have default settings too.

The display is a 14" 1980x1080 on the laptop if that matters. I can plug a QHD (2560x1440) 27" at work for test if necessary too.
